kuflow-engine-client-activity-api-task/src/main/java/com/kuflow/engine/client/activity/api/task/service/KuFlowActivities.java

```diff
@@ -54,7 +57,20 @@
      * @return task created
      */
     @ActivityMethod
-    TaskResponseResource createTask(TaskRequestResource request);
+    @Nonnull
+    TaskResponseResource createTask(@Nonnull TaskRequestResource request);
+
+    /**
+     * Create a Task and optionally fill its elements and wait that the task reach "COMPLETED" or "CANCELLED" state.
```

Comment:
* Create a Task and optionally fill its elements. The activity is not completed until the "COMPLETED" or "CANCELLED" event is received from KuFlow. This is useful in KuFlow tasks where you have to wait for an external agent, usually a human, to complete it.

Reply:
done